.github/actions/py-dependencies/action.yaml

```diff
@@ -109,8 +109,17 @@ runs:
         (. ${VIRTUALENV_DIR}/bin/activate; cd ${ST2_REPO_PATH}/st2common; python setup.py develop)
         echo "::endgroup::"
 
+    - name: Extract Pack supported python_versions
+      if: inputs.mode == 'pack'
+      shell: bash
+      working-directory: pack
+      run: |
+        echo "PACK_PYTHON_VERSIONS=$(${CI_DIR}/utils/extract_from_yaml.py pack.yaml python_versions)" >> ${GITHUB_ENV}
+
     - name: Install Pack Requirements
       if: inputs.mode == 'pack'
+      # continue if pack does not support this version of python
+      continue-on-error: ${{ ! contains(fromJSON(env.PACK_PYTHON_VERSIONS), inputs.python-version-short[0]) }}
```

Comment:
I'm afraid indexing a string might not work in gha... not sure how to test that atm.

Reply:
I'll merge, re-run a test to use it, and push a followup if needed